src/NuGet.Clients/NuGet.VisualStudio.Common/OptionsPage.cs

```diff
@@ -6,6 +6,7 @@ namespace NuGet.PackageManagement.VisualStudio
     public enum OptionsPage
     {
         General,
-        PackageSources
```

Comment:
I don't imagine you meant to have this change + the src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/OptionsPageActivator.cs change?